### ISSUE TITLE
Loss became 'Nan' when I train the MUINT using our night2day datasets

### ISSUE THREAD

**Issue by 805300384** [NVlabs/MUNIT#35]
Loss, parameters, and weight of networks became Nan after some time of training MUNIT.  I don't know how it happened. 

I have tried to adjust some hyperparameters, but it seemed of no use. I wanna know how I can make adjust to solve this problem.

I'm not really deep into this topic, so really appreciate your answer.

**Comment by Cuky88** [comment]
Without further details no one can tell you what's wrong. MUNIT works fine with the given settings and parameters.

Since you changed parameters, it would be of help if you post your settings and any other changes. 
Besides that, which datasets are you using?

When I encountered such problems, in the most cases it's either due to too high learning rates or the erroneous data (images) itself, of course if you did not change anything on the code.

**Comment by circlehy** [comment]
same problem, using my own dataset, which has been used for many other algorithm, training loss(trainer.loss_gen_total and trainer.loss_dis_total) turn NaN quickly. Then I changed the lr within 0.0001 to 0.0000001 , still can not fix the problem. I am not sure what maybe wrong with it.

**Comment by 805300384** [comment]
> @805300384 the problem is in dataloader. check what dataloader samples. sometimes it can return samples filled with `1.` which leads to NaNs

Thank you very much.

**Comment by 805300384** [comment]
> same problem, using my own dataset, which has been used for many other algorithm, training loss(trainer.loss_gen_total and trainer.loss_dis_total) turn NaN quickly. Then I changed the lr within 0.0001 to 0.0000001 , still can not fix the problem. I am not sure what maybe wrong with it.

I changed my dataset and the problem was solved

**Comment by syz825211943** [comment]
So how to solve this problem? Could you give me some suggestions? Thanks!
